RELEASE_NOTES.rst

```diff
@@ -8919,7 +8919,7 @@ New Webserver UI with Role-Based Access Control
 
 The current webserver UI uses the Flask-Admin extension. The new webserver UI uses the `Flask-AppBuilder (FAB) <https://github.com/dpgaspar/Flask-AppBuilder>`_ extension. FAB has built-in authentication support and Role-Based Access Control (RBAC), which provides configurable roles and permissions for individual users.
 
-To turn on this feature, in your airflow.cfg file (under [webserver]), set the configuration variable ``rbac = True``\ , and then run ``airflow`` command, which will generate the ``webserver_config.py`` file in your $AIRFLOW_HOME.
+To turn on this feature, in your airflow.cfg file (under [webserver]), set the configuration variable ``rbac = True``\ , and then run ``airflow`` command, which will generate the ``webserver_config.py`` file in your $AIRFLOW_HOME/config.
```

Comment:
I don’t think this should be changed. This line describes things _when 1.10.0 was released_ and history should not be altered. Also I believe this feature no longer exists.

Reply:
Good idea @uranusjr.
Moved the content to a newsfragment file. Can you look at it again?